test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs

```diff
@@ -102,9 +104,11 @@ IEnumerable<Registry> GetAndValidateRegistries()
             this.LoadGenImage = Option.Maybe(Get("loadGenImage"));
             this.RelayerImage = Option.Maybe(Get("relayerImage"));
             this.NetworkControllerImage = Option.Maybe(Get("networkControllerImage"));
+            this.PlugAndPlayIdentityImage = Option.Maybe(Get("plugAndPlayIdentityImage"));
             this.EdgeAgentBootstrapImage = Option.Maybe(Get("edgeAgentBootstrapImage"));
             this.TestTimeout = TimeSpan.FromMinutes(context.GetValue("testTimeoutMinutes", 5));
             this.Verbose = context.GetValue<bool>("verbose");
+            this.IoTHubAccessToken = Option.Maybe(Get("HUB_ACCESS_TOKEN"));
```

Comment:
Is `IoTHubAccessToken` begin used anywhere? It isn't documented in the README and I don't see any references...